frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/textProperty.component.tsx

```diff
@@ -16,11 +16,11 @@
  */
 
 import { PropertyDefinition } from '@/v5/store/tickets/tickets.types';
-import { TextField } from '@mui/material';
+import { FormTextField } from '@controls/formTextField/formTextField.component';
 
-export const TextProperty = ({ property, value }: {property: Partial<PropertyDefinition>, value:any}) => (
-	<div>
-        &nbsp;
-		<TextField label={property.name} value={value} disabled={property.readOnly} />
-	</div>
+export const TextProperty = ({
+	property: { name, readOnly, required },
```

Comment:
Same question

Reply:
reference [this answer](https://github.com/3drepo/3drepo.io/pull/3696#discussion_r1001789280)